Realm/Tests/AsyncTests.mm

```diff
@@ -414,11 +414,7 @@ - (void)testErrorHandling {
     XCTestExpectation *exp = [self expectationWithDescription:@""];
     auto token = [IntObject.allObjects addNotificationBlock:^(RLMResults *results, RLMCollectionChange *change, NSError *error) {
         XCTAssertNil(results);
-        XCTAssertNotNil(error);
-        // Validate error
-        XCTAssertEqualObjects(error.domain, RLMErrorDomain);
-        XCTAssertEqual(error.code, RLMErrorFail);
-        XCTAssert([error.localizedDescription rangeOfString:@"Too many open files"].location != NSNotFound);
+        RLMValidateRealmError(error, RLMErrorFail, @"Too nany open files");
```

Comment:
This test was not failing unfortunately, but I fixed the macro and the typo and made sure it worked with both good and bad test cases.